src/components/Missions/MissionsPage.js

```diff
@@ -1,16 +1,32 @@
 const MissionsPage = (prop) => {
-  const { name, description } = prop;
+  const {
+    joinMissionHandler, id, name, description, reserved, leaveMissionHandler,
```

Comment:
Like I said in the previous comment, you can declare here your functions without passing as props and you don't need to change anything on your onClick event 

> const joinMissionHandler = (id) => {
>     dispatch(joinMissions(id));
>   };
> 
>   const leaveMissionHandler = (id) => {
>     dispatch(leaveMissions(id));
>   };